docs/usage/logging.md

```diff
@@ -0,0 +1,92 @@
+# Logging
+
+## Overview
+
+This small text provides instructions on how to configure logging using JSON and INI files. The logging configuration is divided into two parts:
```

Comment:
```suggestion
Instructions on how to configure logging using JSON and INI files. The logging configuration is divided into two parts:
```

Reply:
Maybe add a sentence describing who this is meant for? (I.e., do we have to worry about this, or can we ignore logging for a first usage?)